cmd/serverless/main.go

```diff
@@ -124,8 +128,9 @@ func runAgent(tagger tagger.Component) {
 	wg.Add(3)
 
 	go startTraceAgent(&wg, lambdaSpanChan, coldStartSpanId, serverlessDaemon, tagger, rcService)
+
```

Comment:
Nit. If you happen to make another commit before merging, can you remove the two empty lines added?

Reply:
Removed 👍